libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/overlay/WaymarkOverlay.kt

```diff
@@ -30,6 +29,7 @@ class WaymarkOverlay(private val mapView: CycleMapView) : Overlay(), PauseResume
     private val screenPos = Point()
     private val bitmapTransform = Matrix()
     private val bitmapPaint = Paint()
+    private val boldTextBrush = Brush.createBoldTextBrush((offset(mapView.getContext())*0.8).toInt())
```

Comment:
another magic number `0.8` here - what does this mean?

Reply:
Text size.